pkg/meta/redis.go

```diff
@@ -118,10 +117,10 @@ func newRedisMeta(driver, addr string, conf *Config) (Meta, error) {
 	if opt.MaxRetries == 0 {
 		opt.MaxRetries = -1 // Redis use -1 to disable retries
 	}
-	opt.MinRetryBackoff = query.duration("min-retry-backoff", time.Millisecond*20)
-	opt.MaxRetryBackoff = query.duration("max-retry-backoff", time.Second*10)
-	opt.ReadTimeout = query.duration("read-timeout", time.Second*30)
-	opt.WriteTimeout = query.duration("write-timeout", time.Second*5)
+	opt.MinRetryBackoff = minRetryBackoff
```

Comment:
we'd better set it only if it's default value, i.e. not set by Redis recognized query string